pgjdbc/src/main/java/org/postgresql/core/v3/QueryExecutorImpl.java

```diff
@@ -2599,7 +2599,7 @@ public void receiveParameterStatus() throws IOException, SQLException {
     if (name.equals("client_encoding")) {
       if (allowEncodingChanges) {
         if (!value.equalsIgnoreCase("UTF8") && !value.equalsIgnoreCase("UTF-8")) {
-          LOGGER.log(Level.WARNING,
+          LOGGER.log(Level.FINE,
```

Comment:
+1. allowEncodingChanges is `false` by default, so user likely knows there's non-default encoding in use.